indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/PartialSegmentGenerateTask.java

```diff
@@ -246,7 +249,8 @@ private Map<String, TaskReport> getTaskCompletionReports()
                 getTaskCompletionRowStats(),
                 "",
                 false, // not applicable for parallel subtask
-                segmentAvailabilityWaitTimeMs
+                segmentAvailabilityWaitTimeMs,
+                intervals
```

Comment:
The subtasks don't need to condense intervals, because the parent task gathers all intervals from subtasks, then condenses them. See ParallelIndexSupervisorTask line 1249.